### PR TITLE
Milestone3: Handle eventsevents

### PR DESCRIPTION
**Pull request by algerina**
In this Milestone:
- I added the files calculate.js and operate.js.
- Imported the files in the Calculator component.
- Implemented the event handlers.

### CODE REVIEW

**Review by sinansevgi — CHANGES_REQUESTED** [review]
`Fixed`

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,35 +1,46 @@
 import React from 'react';
 import '../calculator.css';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      total: 0,
```
Comment (sinansevgi) [inline src/components/calculator.js]: - [x] For this milestone, we are expecting you to practice what you have learned until now; like passing props to child components, using lifecycle methods, lifting state up. You should use these concepts in your app.

Tip:
_You should divide your `Calculator` component into child components in order to practice these concepts. (For example, you can create a display component for displaying calculations and a keypad component for the keys.)_



From the project requirements ⬇️ 
>Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift the state up.

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @algerina, 

Good job so far!
The calculator works like a charm :clap:
There are some issues that you still need to work on to go further but you are almost there!


 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by algerina** [comment]
> # Changes Required 
> Hi @algerina,
> 
> Good job so far! The calculator works like a charm clap There are some issues that you still need to work on to go further but you are almost there!
> 
> ## Required Changes 
> _Please check the inline comments for my suggestions_ arrow_down
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Happy Hacking! computer
> 
> You can reply to this conversation if you have any questions or need elaboration. wave Sinan Sevgi
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@sinansevgi Thank you for your review, but I can't see the required changes or the comments.

File: src/components/calculator.js (hunk 1 above)
Comment (algerina) [reply src/components/calculator.js]: @sinansevgi Thank you for your Feedback but can you give further explanation, please.

**Comment by sinansevgi** [comment]
> @sinansevgi Thank you for your review, but I can't see the required changes or the comments.

Hi @algerina,
I noticed that you were pushing commits while I was reviewing the project so I updated my review. Please check the second review

**Review by Emmyn5600 — APPROVED** [review]
STATUS: APPROVED :+1: 

Hello, the team @GintsM  👋🏾 👋🏾 congratulations on making it so far on this milestone.

## Optional suggestions

- n/a

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

**Pull request merged**